lib/vet360/contact_information/service.rb

```diff
@@ -18,6 +18,26 @@ def get_person
       rescue StandardError => e
         handle_error(e)
       end
+
+      def post_email(vet360_email)
+        post_or_put_email(:post, vet360_email)
+      end
+
+      def put_email(vet360_email)
+        post_or_put_email(:put, vet360_email)
+      end
+
+      private
+
+      def post_or_put_email(method, vet360_email)
+        with_monitoring do
+          raw = perform(method, "emails", vet360_email.to_request)
+          Vet360::ContactInformation::EmailUpdateResponse.new(raw.status, Vet360::Models::Email.from_response(raw))
```

Comment:
You'll want to pass in the raw response here so we can keep a record of what it was in addition to the parsed response data we're building.